.github/workflows/quick-checks.yml

```diff
@@ -0,0 +1,26 @@
+name: Quick Checks
+
+on:
+  - push
+
+jobs:
+  quick-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Run quick checks
+        run: mvn -B clean verify --file pom.xml -DskipTests
```

Comment:
-DskipTests: I assume the tests are disabled intentionally (maybe because you don't have the database?)?

Reply:
yes, here I just run all the other checks as a quick check, since quite often for example project-keeper breaks the build. In those cases it's better if we see it right here, instead of after starting the cluster which took 40 mins.